src/webview.cpp

```diff
@@ -71,6 +72,24 @@ void WebView::wheelEvent(QWheelEvent *event) {
     }
 }
 
+void WebView::contextMenuEvent(QContextMenuEvent *event)
+{
+    auto menu = this->page()->createStandardContextMenu();
+    pageAction(QWebEnginePage::OpenLinkInNewWindow)->setVisible(false);
```

Comment:
This should be set only once at application initialization.

Reply:
It will not work this way. I use this method to get the default contextmenu that Qt will use (it is different depending on the situation) and then add the action